### ISSUE TITLE
make build error on Ubuntu 22.04

### ISSUE THREAD

**Issue by snowyu** [josStorer/RWKV-Runner#207]
on master branch(0005816c1d0432444c51421ecc61fc51ddfa3843)

```bash
cd RWKV-Runner
git pull
make build
# Building target: linux/amd64

  • Generating bindings: Done.
  • Installing frontend dependencies: Done.
  • Compiling frontend:     
    > frontend@0.0.0 build
    > tsc && vite build
    
    src/pages/Train.tsx(10,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslCommand'.
    src/pages/Train.tsx(11,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslEnable'.
    src/pages/Train.tsx(12,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslInstallUbuntu'.
    src/pages/Train.tsx(13,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslIsEnabled'.
    src/pages/Train.tsx(14,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslStart'.
    src/pages/Train.tsx(15,3): error TS2305: Module '"../../wailsjs/go/backend_golang/App"' has no exported member 'WslStop'.
    src/pages/Train.tsx(333,16): error TS7006: Parameter 'e' implicitly has an 'any' type.
    src/pages/Train.tsx(348,14): error TS7006: Parameter 'e' implicitly has an 'any' type.
  ERROR   exit status 2
```

**Comment by josStorer** [comment]
```
rm ./backend-golang/wsl.go
mv ./backend-golang/wsl_not_windows.go ./backend-golang/wsl.go
```

https://github.com/josStorer/RWKV-Runner/blob/master/.github/workflows/release.yml#L115-L116

**Comment by snowyu** [comment]
Tnks. Why not use to conditional compilation?  
https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

**Comment by josStorer** [comment]
Already fixed through 1d6aeab9dc121b167edfb671cc16aac8a394ac50, now just execute `make` command.

I previously had added `//go:build` conditional compilation to the `wsl*.go` file, but found that it was not working properly on Linux. I initially thought it was a bug in Wails and temporarily fixed the issue by manually replacing the file in the build pipeline. Then I put it aside.

Recently, I checked and found that the issue was due to a naming mistake of the `wsl_not_windows.go` file on Linux. Because the file name ended with `windows`, go compiler ignored this file and did not correctly export the JavaScript bindings for Linux.
